src/main/java/com/flipkart/rasl/App.java

```diff
@@ -0,0 +1,69 @@
+package com.flipkart.rasl;
+
+import org.apache.commons.dbcp.ConnectionFactory;
+import org.apache.commons.dbcp.DriverManagerConnectionFactory;
+import org.apache.commons.dbcp.PoolableConnectionFactory;
+import org.apache.commons.dbcp.PoolingDataSource;
+import org.apache.commons.pool.impl.GenericObjectPool;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.*;
+
+
+import org.apache.commons.dbcp.ConnectionFactory;
+import org.apache.commons.dbcp.DriverManagerConnectionFactory;
+import org.apache.commons.dbcp.PoolableConnectionFactory;
+import org.apache.commons.dbcp.PoolingDataSource;
+import org.apache.commons.pool.impl.GenericObjectPool;
+import org.apache.tomcat.jdbc.pool.DataSource;
+import org.json.JSONArray;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
+{
+
+
+    public static void main( String[] args ) throws Exception {
+
+        List<Integer> range_start = Arrays.asList(220000000,235419296,220000000,220000000);
```

Comment:
Read this value from config, try to fetch this value from Database in version 2.